test/sai_test/config/lag_configer.py

```diff
@@ -42,6 +42,7 @@ def t0_lag_config_helper(test_obj: 'T0TestBase', is_create_lag=True):
     if is_create_lag:
         test_obj.dut.lag1 = lag_configer.create_lag([17, 18])
         test_obj.dut.lag2 = lag_configer.create_lag([19, 20])
+        test_obj.dut.lag3 = lag_configer.create_lag([21, 22])
```

Comment:
we have more than three lags, maybe add them to a list can make other operation on them easier